airflow_prometheus_exporter/prometheus_exporter.py

```diff
@@ -277,6 +278,57 @@ def get_task_duration_info():
         )
 
 
+def get_landing_times():
+    """Duration of successful tasks in seconds."""
+    with session_scope(Session) as session:
+        max_execution_dt_query = (
+            session.query(
+                DagRun.dag_id,
+                func.max(DagRun.execution_date).label("max_execution_dt"),
+            )
+            .join(DagModel, DagModel.dag_id == DagRun.dag_id,)
+            .filter(
+                DagModel.is_active == True,  # noqa
```

Comment:
Is it necessary to put the `==` given that `is_active` and `is_paused` are already booleans?

Reply:
Yes, i just tested.
I did it like that because i was just copying the query from above.
Without that renders something like
`.... WHERE False AND DagRun.end_date IS NOT NULL ...`